packages/contracts-bedrock/src/L1/DelayedVetoable.sol

```diff
@@ -133,18 +150,18 @@ contract DelayedVetoable is ISemver {
     function _handleCall() internal {
         // The initiator and vetoer activate the delay by passing in null data.
```

Comment:
nit: can cache calls onto the stack instead of making the same calls multiple times

Reply:
Good call.
This pattern was safer and more iterative during refactoring, but caching finishes the job.